### DISCUSSION TITLE
Can I run a master and volume node on a single server?

### DISCUSSION THREAD

**Discussion by jmg-duarte** [seaweedfs/seaweedfs#5487] [Q&A] [answered]
I know that this question sounds stupid at first. But hear me out!

I'm currently building a service on a 0-budget for the PoC, hence, I'm self hosting everything on my personal server. I looked into MinIO, Ceph and others but SeaweedFS seems to be the simpler one.
I'm looking to store pictures I'm scraping off of the internet, to later display on a mobile app (I can't just host the links to the original pictures because they may disappear).

On the README it states:
> Ideally, they should be started from different machines. 

So my question boils down to _how ideal?_

If it is a matter of fault tolerance, I'm ok running it out of a single place. 
If it is a matter of performance, it might depend as I'm already hosting the server and PostgreSQL out of the server, but that load is almost negligible so it doesn't seem much of an issue either.

Thank you!

**Comment by chrislusf** [comment, marked as answer]
yes. You can use the "weed server" command, or "weed server -s3".

"ideal" is for performance and tolerance.
